__tests__/components.test.js

```diff
@@ -124,9 +126,14 @@ describe('Components', () => {
       [/block]`,
       rdmd: `[](https://www.nytimes.com/2020/05/03/us/politics/george-w-bush-coronavirus-unity.html "@embed")`,
     };
-    Object.values(fixtures).map(fx => {
-      const wrap = mount(markdown.react(fx));
-      return expect(wrap.html()).toMatchSnapshot();
+
+    silenceConsole(({ error }) => {
```

Comment:
Can you explain this to me a bit? Is it to test for things we _know_ will cause errors?

Reply:
Yes. Originally I just wanted to silence the errors so there would be less noise in the test results. Then I figured I should at least check that it's only complaining the once. But, that also doesn't seem that relevant.

Reply:
s'okay by me—just wanted to understand!